examples/multi-pointer.c

```diff
@@ -86,7 +86,7 @@ static void handle_output_add(struct output_state *ostate) {
 
 		struct wlr_xcursor_image *image = sample->xcursor->images[0];
 		wlr_cursor_set_image(cursor->cursor, image->buffer, image->width,
-			image->width, image->height, image->hotspot_x, image->hotspot_y);
+			image->width, image->height, image->hotspot_x, image->hotspot_y, 0);
```

Comment:
Scale factor of 0? What?

Reply:
https://github.com/swaywm/wlroots/pull/412/files#diff-69dfd47b5b7a981f4364342fc4a54ed0R67

This basically allows compositors to do not care about cursor scaling if they don't want to.

Reply:
Normally this is indicated by -1.

Reply:
Since it's an `uint32_t` and since zero hasn't any meaning for a scale, I thought it would be okay. I can change it to an `int32_t` though.

Reply:
I don't care either way.